src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,492 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Preload all video for faster playback.
+
+{# wf_published_on: 2017-06-21 #}
+{# wf_updated_on: 2017-06-21 #}
+
+# Fast Playback with Video Preload {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque suscipit
+nibh nec feugiat ornare. Nam tristique, magna non pharetra mollis, diam elit
+faucibus nibh, ut condimentum diam metus vel dolor. Integer facilisis magna at
+ex bibendum luctus. In eu tincidunt ipsum. Duis scelerisque, tortor quis
+condimentum finibus, est neque mollis leo, at imperdiet justo elit dapibus
+nisl. Integer gravida dapibus turpis, vel scelerisque dolor scelerisque eget.
+Sed dolor lectus, tempor vulputate bibendum eget, accumsan lobortis ex.
+
+<video controls controlsList="nodownload" muted playsinline style="width: 100%">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.webm#t=0.7"
+          type="video/webm">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.mp4#t=1"
+          type="video/mp4">
```

Comment:
@johnpallett suggested I should not use YouTube there. Yes, I've manually modified code to make it look slower to load.